src/main/java/baseEntities/BaseTest.java

```diff
@@ -0,0 +1,31 @@
+package baseEntities;
```

Comment:
Please store BaseTest class in the directory with all the tests

Reply:
done